backend/app/GraphQL/Queries/Question.php

```diff
@@ -31,10 +33,21 @@ public function __invoke($_, array $args)
 
         $question = $questionQuery->where('slug', $args['slug'])->first();
 
-        if ($args['shouldAddViewCount']) {
-            $question->update(['views_count' => $question->views_count + 1]);
+        $checker = true;
+
+        if (! $question->is_public) {
+            $team = Team::findOrFail($question->team_id);
```

Comment:
Please use try and catch when using findOrFail so the fail will be captured with custom exceptions that return the correct error message